core/unit_test/CMakeLists.txt

```diff
@@ -24,200 +24,108 @@ KOKKOS_TARGET_INCLUDE_DIRECTORIES(kokkos_gtest PUBLIC ${GTEST_SOURCE_DIR})
-      threads/TestThreads_ViewMapping_b.cpp
-      threads/TestThreads_ViewMapping_subview.cpp
-      threads/TestThreads_ViewOfClass.cpp
-      threads/TestThreads_Crs.cpp
-      threads/TestThreads_WorkGraph.cpp
-      threads/TestThreads_ViewResize.cpp
+    SOURCES ${Threads_SOURCES}
   )
 ENDIF()
 
 IF(Kokkos_ENABLE_OPENMP)
   KOKKOS_ADD_EXECUTABLE_AND_TEST(
     UnitTest_OpenMP
     SOURCES
-      UnitTestMainInit.cpp
-      openmp/TestOpenMP_AtomicOperations_int.cpp
-      openmp/TestOpenMP_AtomicOperations_unsignedint.cpp
-      openmp/TestOpenMP_AtomicOperations_longint.cpp
-      openmp/TestOpenMP_AtomicOperations_unsignedlongint.cpp
-      openmp/TestOpenMP_AtomicOperations_longlongint.cpp
-      openmp/TestOpenMP_AtomicOperations_double.cpp
-      openmp/TestOpenMP_AtomicOperations_float.cpp
-      openmp/TestOpenMP_AtomicOperations_complexdouble.cpp
-      openmp/TestOpenMP_AtomicOperations_complexfloat.cpp
-      openmp/TestOpenMP_AtomicViews.cpp
-      openmp/TestOpenMP_Atomics.cpp
-      openmp/TestOpenMP_Complex.cpp
-      openmp/TestOpenMP_Init.cpp
-      openmp/TestOpenMP_MDRange_a.cpp
-      openmp/TestOpenMP_MDRange_b.cpp
-      openmp/TestOpenMP_MDRange_c.cpp
-      openmp/TestOpenMP_MDRange_d.cpp
-      openmp/TestOpenMP_MDRange_e.cpp
-      openmp/TestOpenMP_Other.cpp
-      openmp/TestOpenMP_RangePolicy.cpp
-      openmp/TestOpenMP_Reductions.cpp
-      openmp/TestOpenMP_Reducers_a.cpp
-      openmp/TestOpenMP_Reducers_b.cpp
-      openmp/TestOpenMP_Reducers_c.cpp
-      openmp/TestOpenMP_Reducers_d.cpp
-      openmp/TestOpenMP_Scan.cpp
-      openmp/TestOpenMP_SharedAlloc.cpp
-      openmp/TestOpenMP_SubView_a.cpp
-      openmp/TestOpenMP_SubView_b.cpp
-      openmp/TestOpenMP_SubView_c01.cpp
-      openmp/TestOpenMP_SubView_c02.cpp
-      openmp/TestOpenMP_SubView_c03.cpp
-      openmp/TestOpenMP_SubView_c04.cpp
-      openmp/TestOpenMP_SubView_c05.cpp
-      openmp/TestOpenMP_SubView_c06.cpp
-      openmp/TestOpenMP_SubView_c07.cpp
-      openmp/TestOpenMP_SubView_c08.cpp
-      openmp/TestOpenMP_SubView_c09.cpp
-      openmp/TestOpenMP_SubView_c10.cpp
-      openmp/TestOpenMP_SubView_c11.cpp
-      openmp/TestOpenMP_SubView_c12.cpp
-      openmp/TestOpenMP_SubView_c13.cpp
-      openmp/TestOpenMP_Task.cpp
-      openmp/TestOpenMP_Team.cpp
-      openmp/TestOpenMP_TeamReductionScan.cpp
-      openmp/TestOpenMP_ViewAPI_a.cpp
-      openmp/TestOpenMP_ViewAPI_b.cpp
-      openmp/TestOpenMP_ViewAPI_c.cpp
-      openmp/TestOpenMP_ViewAPI_d.cpp
-      openmp/TestOpenMP_ViewAPI_e.cpp
-      openmp/TestOpenMP_ViewMapping_a.cpp
-      openmp/TestOpenMP_ViewMapping_b.cpp
-      openmp/TestOpenMP_ViewMapping_subview.cpp
-      openmp/TestOpenMP_ViewOfClass.cpp
-      openmp/TestOpenMP_Crs.cpp
-      openmp/TestOpenMP_WorkGraph.cpp
-      openmp/TestOpenMP_UniqueToken.cpp
-      openmp/TestOpenMP_ViewResize.cpp
+    ${OpenMP_SOURCES}
+    openmp/TestOpenMP_Task.cpp
+    openmp/TestOpenMP_UniqueToken.cpp
```

Comment:
Why does the `UniqueToken` get special treatment?
Is there a good reason for not running it in serial and with pthreads?

Reply:
Seems to work.